### DISCUSSION TITLE
Using pocketbase for Desktop Application embedding FE code, html and css in the SQLite DB

### DISCUSSION THREAD

**Discussion by alucab** [pocketbase/pocketbase#4164] [Q&A]
As from the title i would like to serve html and css as part of the sqlite db to have a compact application made of only two files
I understood that i have to manage and customize the static routing but i would like to have a couple of suggestions to know where to start looking into the code

Thanks!

**Comment by ganigeorgiev** [comment]
I'm not sure about embedding assets in your SQLite db file but for embedding static assets in your binary you can use the [Go embed directive](https://pkg.go.dev/embed).
As an example you can check the [`ui` package](https://github.com/pocketbase/pocketbase/tree/master/ui) (the Admin UI static Svelte app) and more specifically [embed.go](https://github.com/pocketbase/pocketbase/blob/master/ui/embed.go).
